src/app/config/config.ts

```diff
@@ -214,6 +214,7 @@ const config: Config = {
   bounceLifeSpan: basicVars.mail.bounceLifeSpan,
   chromiumBin: basicVars.mail.chromiumBin,
   port: basicVars.core.port,
+  corsWhiteList: basicVars.corsWhitelist,
```

Comment:
can we standardise this to capital L or small l pls?

Reply:
good catch, haha will fix